sonarqube-sample-java/src/main/java/com/consoleconnect/mw/sonarqube/sample/service/UserService.java

```diff
@@ -27,6 +28,14 @@ public User create(String email, String password) {
 		user.setPassword(password);
 
 		email2User.put(email, user);
+
+		while(true) {
+			//do it
+		}
+		if (tmp.equals("tmp")) {
+			System.out.println("tmp is tmp");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS106)